test/integration/targets/loops/tasks/main.yml

```diff
@@ -27,20 +27,27 @@
   block:
     - name: Measure time before loop with .5s pause
       set_fact:
-        times: "{{times|default([]) + [ lookup('pipe','date +%s.%3N') ]}}"
+        times: "{{times|default([]) + [ now(fmt='%s.%f') ]}}"
       with_sequence: count=3
       loop_control:
         pause: 0.6
 
-    - name: ensure lag, since there is 3 rounds, and 0.5 seconds between, it should last 1.2 seconds, but allowing leeway due to CI lag
-      assert:
-        that:
-            - tdiff|float >= 1.2
-            - tdiff|int < 3
-      vars:
+    - name: Debug times var
+      debug:
+        var: times
```

Comment:
Do we still need this in the final test? Maybe it was just for troubleshooting?

Reply:
I prefer adding debugs, when we do assertions on data.  Without it, you cannot directly troubleshoot the variables, without modifying the playbook.  So it wasn't accidental, it's to make it easier to troubleshoot.